apps/tup-cms/src/taccsite_custom/tup_cms/static/tup_cms/css/for-tup-cms/components/c-footer.css

```diff
@@ -17,7 +17,7 @@ svg {
 .c-footer .c-button,
 .c-footer [class*="button--"] {
     --max-width: auto; /* override core-styles.base.css */
-    font-size: 80%; /* mimic Botstrap .small and <small> */
+    font-size: 10px; /* mimic Botstrap .small and <small> */
```

Comment:
```suggestion
    font-size: var(--global-font-size--x-small); /* mimic Botstrap .small and <small> */
```

| use var | check size |
| - | - |
| ![use var](https://github.com/TACC/tup-ui/assets/62723358/f83f17f6-9c9f-44bb-b675-43d98f1717c4) | ![check size](https://github.com/TACC/tup-ui/assets/62723358/e1020900-1d6c-466f-a413-8a6e609450c7) |